config/routes.rb

```diff
@@ -1,11 +1,12 @@
 IntergalacticBattleship::Application.routes.draw do
+  devise_for :users
   root "strikes#index"
   resources :strikes, only: [:new, :create]
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'static_pages#index'
```

Comment:
I think it means you deletes the line in `config/routes.rb` and that it's causing problems.  It looks like someone else set up an extra root line at `strikes#index` above but prior to this PR it wasn't doing anything.

I think we want to delete line 2 and add back in line 8, unless I'm missing something.